src/main/java/racingcar/model/RacingGame.java

```diff
@@ -0,0 +1,32 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGame {
+
+    private Cars cars;
+    private List<Car> raceResult;
+    private TryCount tryCount;
+    private List<RaceResult> totalResult;
```

Comment:
`raceResult`와 `totalResult`는 RacingGame 객체의 상태로 둘 필요가 없는 것 같아요. 🤔 
먼저 `totalResult`는 **startRace** 메서드 안의 지역변수로도 충분히 대체할 수 있을 것 같아요. 😃 
그리고 `raceResult`는 `getFinalRoundResult` 메서드의 결과 값으로 전달하는데 이때 `cars` 객체를 전달해도 같은 결과가 나올 것 같아요. 👀 
한 번, 변경해보시고 잘 이해가 안되는 부분이 있으면 언제든지 DM주세요! 🙏

```suggestion
    private TryCount tryCount;
```